packages/vue-cli-plugin/generator/index.js

```diff
@@ -1,23 +1,55 @@
+const insertVuesticPlugin = (lines) => {
+  /* Vue app stored in variable
+  const app = createApp(App)
+  ...
+  app.mount('#app')
+  */
+  const renderIndex = lines.findIndex(line => line.match(/app\.mount/))
+
+  if (renderIndex !== -1) {
+    lines[renderIndex - 1] += `\n\napp.use(VuesticPlugin);`
+
+    return lines
+  } else {
+    /* Vue app chained
+    createApp(App).mount('#app')
+    */
+    return lines.map((line) => {
+      if (line.match(/\.mount/) != null) {
+        return line.replace('.mount', '.use(VuesticPlugin).mount')
+      }
+      
+      return line
+    })
+  }
+}
+
 module.exports = (api, options) => {
   const fs = require('fs')
   const { EOL } = require('os')
 
   api.extendPackage({
     dependencies: {
-      'vuestic-ui': require('../../ui/package').version,
```

Comment:
I don't know how this should work. For me, this requirement blocked me from[ local installation](https://cli.vuejs.org/dev-guide/plugin-dev.html#installing-plugin-locally), so I just hardcoded a specific version.